### PR TITLE
Fix #763, clarification in section 9.1.4

### LINKED ISSUE #763: Ordering of profiles in Section 9.1.4
The example in section 9.1.4 says:
> [IA Sequence Header OBU](https://aomediacodec.github.io/iamf/#ia-sequence-header-obu): take the larger [primary_profile](https://aomediacodec.github.io/iamf/#primary_profile) field and the larger [additional_profile](https://aomediacodec.github.io/iamf/#additional_profile) field from the two input [IA Sequence](https://aomediacodec.github.io/iamf/#ia-sequence)s.

We should not consider profiles to be ordered. It should talk about determining the profile corresponding to the merged result. 
Also the 2 sequences are supposed to be compliant to the Simple Profile so probably the primary profile is set to Simple so the determination is simple.

I suggest rewording that part.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/783.html" title="Last updated on Nov 29, 2023, 1:15 AM UTC (c908642)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/783/cb5b4b2...c908642.html" title="Last updated on Nov 29, 2023, 1:15 AM UTC (c908642)">Diff</a>

### CODE REVIEW

**Review by felicialim — APPROVED** [review]

**Review by cconcolato — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -3084,7 +3084,7 @@ This section provides a way to generate an [=IA Sequence=] with two [=Audio Elem
 This section provides a way to generate an [=IA Sequence=] with two [=Audio Element=]s from two [[#profiles-simple|Simple Profile]] [=IA Sequence=]s with the same [=Codec Config OBU=]. The result complies with the [[#profiles-base|Base Profile]].
 
 Step 1: [=Descriptors=] are generated as follows:
-- [=IA Sequence Header OBU=]: take the larger [=primary_profile=] field and the larger [=additional_profile=] field from the two input [=IA Sequence=]s.
+- [=IA Sequence Header OBU=]: Both [=primary_profile=] and [=additional_profile=] fields are set to 1 to indicate [[#profiles-base|Base Profile]].
```
Comment (felicialim) [inline index.bs:3087]:
```suggestion
- [=IA Sequence Header OBU=]: Both [=primary_profile=] and [=additional_profile=] fields are set to 1 to indicate the [[#profiles-base|Base Profile]].
```

**Review by felicialim — APPROVED** [review]
